spec/unit/shoulda/matchers/active_record/association_matcher_spec.rb

```diff
@@ -1117,4 +1137,13 @@ def define_association_with_order(model, macro, name, order, other_options={})
     args << options
     model.__send__(macro, name, *args)
   end
+
+  def dependent_options(rails_version)
```

Comment:
What do you think about this method simply referring to `Rails.version` directly instead of taking it as an argument?